tests/duty.test.ts

```diff
@@ -0,0 +1,144 @@
+import request from 'supertest';
+import dotenv from 'dotenv';
+import lodash from 'lodash';
+import { Collection, ObjectId } from 'mongodb';
+import { Express } from 'express-serve-static-core';
+import { Duty, DutySchema } from '../services/schemas/duty.zschema';
+import { getServer } from '../app';
+import { getDutiesCollection } from '../db_connection';
+import createTestDuties from './duty.data';
+
+dotenv.config();
+
+let server: Readonly<Express>;
+let collection: Readonly<Collection<Duty>>;
+
+const testDuties = createTestDuties(3);
+
+beforeAll(async () => {
+  collection = await getDutiesCollection();
+  server = getServer();
+});
+
+beforeEach(async () => {
+  await collection.deleteMany({});
+});
+
+afterAll(async () => {
```

Comment:
You are using `afterAll` and `afterEach`, but both of them do lthe same thing

Reply:
Where? I'm doing `beforeEach` and `afterAll`

Reply:
resolved

Reply:
I meant `afterAll` and `beforeEach`

Reply:
OK, so what is the problem? They aren't doing the same thing